### PR TITLE
ci: grab deploy info an archive it

### PR DESCRIPTION
**Pull request by kuisathaverat**
* Archive a file packages.txt with the versions of the packages
* Archive a file distribution_version.json with some info of the Docker image deployed
* Send links to those files in the Slack message
* Send the person who triggers the job in the slack message

### CODE REVIEW

File: .ci/release.groovy [hunk 1]
```diff
@@ -68,10 +68,34 @@ pipeline {
...
+        ''')
+        archiveArtifacts(allowEmptyArchive: true, artifacts: 'packages.txt,distribution_version.json', onlyIfSuccessful: true)
+      }
+    }
   }
   post {
     success {
-      slackMessage(statusMessage: "${params.environment} package storage cluster deployed successfully", color: 'good')
+      slackMessage(statusMessage: "${params.environment} package storage cluster deployed successfully\n"
+        + "User: ${env?.BUILD_CAUSE_USER ? env.BUILD_CAUSE_USER : 'Unknown'}\n"
```
Comment (ycombinator) [inline .ci/release.groovy:96]: I wonder if we should post this in the #integrations channel instead? We already post notifications when a registry distribution is deployed.

Comment (kuisathaverat) [reply .ci/release.groovy:96]: The comment is posted in the #integration channel https://github.com/elastic/package-storage/blob/183774f482127bf5fed85e202ed5a476dfdc5a5c/.ci/release.groovy#L109

**Review by ycombinator — APPROVED** [review]
LGTM.

**Review by EricDavisX — APPROVED** [review]
excellent, thanks! quick win ftw!

**Pull request merged**